apps/admin-gui/src/app/admin/pages/admin-page/admin-ext-sources/admin-ext-sources.component.html

```diff
@@ -1,7 +1,13 @@
 <div>
   <h1 class="page-subtitle">{{'ADMIN.EXT_SOURCES.TITLE' | translate}}</h1>
-  <perun-web-apps-refresh-button (click)="onLoad()"></perun-web-apps-refresh-button>
-
+  <perun-web-apps-refresh-button (click)="refreshTable()"></perun-web-apps-refresh-button>
+  <button
+    (click)="loadConfigExtSources()"
+    color="accent"
+    class="action-button me-2"
+    mat-flat-button>
+    {{'ADMIN.EXT_SOURCES.LOAD' | translate}}
+  </button>
```

Comment:
```suggestion
  <button
    (click)="loadConfigExtSources()"
    class="action-button me-2"
    mat-stroked-button>
    {{'ADMIN.EXT_SOURCES.LOAD' | translate}}
  </button>
```

Reply:
Was meaning to do this as well, but decided against it since I thought you would be mad it's not consistent with the rest of the buttons :D

Reply:
Well the way it was before it was inconsistent so I guess you were right.